com.nickgirga.webready.yml

```diff
@@ -0,0 +1,49 @@
+id: com.nickgirga.webready
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: webready
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # access to home files; for locating WADs and custom installed source ports
+  - --filesystem=home
+
+modules:
+  - name: webready
+    buildsystem: simple
+    build-commands:
+      - install -D webready /app/bin/webready
+      - install -D webready.glade /app/share/webready/webready.glade
+      - install -D res/icons/scalable.svg /app/share/icons/hicolor/scalable/apps/com.nickgirga.webready.svg
+      - install -D res/icons/128x.png /app/share/icons/hicolor/128x128/apps/com.nickgirga.webready.png
```

Comment:
```suggestion
```

svg is enough

Reply:
I was using the 128x PNG as a resource for the about page. Just updated it to create a Pixbuf from the SVG. Will create a new release tag and update the flatpak manifest and stop including the PNG.